recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,4 @@
 c_compiler:
-  - g++
+  - gcc
```

Comment:
Actually gcc the binary is not used, since there is no C code. So this change is not strictly needed. You were already using GCC (the compiler suite).